KmsClient.js

```diff
@@ -701,3 +743,36 @@ function _getRootZcapId({keystoreId, keyId}) {
   }
   return `${ZCAP_ROOT_PREFIX}${encodeURIComponent(suffix)}`;
 }
+
+/**
+ * Creates and signs the http headers for zCap invocation.
+ *
+ * @param {object} options - Options hashmap.
+ * @param {string} options.url - zCap target url.
+ * @param {string} options.method - HTTP method ('get', 'post').
+ * @param {object} [options.headers] - HTTP headers.
+ * @param {object} [options.operation] - Optional JSON payload (for POSTs).
+ * @param {object} [options.capability] - Optional existing capability.
+ * @param {{sign: Function}} options.invocationSigner - Key Signer object.
+ * @param {string} options.capabilityAction - zCap action to perform.
+ *
+ * @returns {Promise<object>} Results with the signed zcap headers object.
+ */
+async function _signCapabilityInvocation({
+  url, method = 'post', headers = this.defaultHeaders, operation, capability,
+  invocationSigner, capabilityAction
+}) {
+  try {
+    // sign HTTP header
+    return signCapabilityInvocation({
```

Comment:
Without an `await` here, the `catch` won't be triggered if an error occurs within any async code in `signCapabilityInvocation`, it will only trigger on JS executed in the same turn of the event loop / microtask queue.

```suggestion
    return await signCapabilityInvocation({
```